src/js/providers/html5.js

```diff
@@ -349,27 +352,31 @@ function VideoProvider(_playerId, _playerConfig, mediaElement) {
             const startDate = _videotag.getStartDate();
             const startDateTime = startDate.getTime ? startDate.getTime() : NaN;
             if (startDateTime !== _this.startDateTime && !isNaN(startDateTime)) {
-                _this.startDateTime = startDateTime;
-                const programDateTime = startDate.toISOString();
-                const { start, end } = _this.getSeekRange();
-                const metadataType = 'program-date-time';
-                const metadata = {
-                    metadataType,
-                    programDateTime,
-                    start,
-                    end
-                };
-                const cue = _this.createCue(start, end, JSON.stringify(metadata));
-                _this.addVTTCue({
-                    type: 'metadata',
-                    cue,
-                });
-                delete metadata.metadataType;
-                _this.trigger(MEDIA_META_CUE_PARSED, { metadataType, metadata });
+                _this.setStartDateTime(startDateTime);
             }
         }
     }
 
+    _this.setStartDateTime = function(startDateTime) {
+        _this.startDateTime = startDateTime;
+        const programDateTime = new Date(startDateTime).toISOString();
```

Comment:
assuming the start date time contains timezone?

Reply:
@jnatalzia https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString